tools/link.py

```diff
@@ -431,10 +431,6 @@ def check_human_readable_list(items):
     extras = settings.BINARYEN_EXTRA_PASSES.split(',')
     passes += [('--' + p) if p[0] != '-' else p for p in extras if p]
 
-  # Run the translator to the standardized EH instructions.
-  if not settings.WASM_LEGACY_EXCEPTIONS:
```

Comment:
I wonder if it would be worthwhile to keep some way to activate the translator path for a while. Just in case, e.g. someone finds a bug, they can try the other pathway?

Reply:
Not sure what would be a good option for enabling this... Adding one more possible value to `WASM_LEGACY_EXCEPTIONS` (which sounds weird because `LEGACY_EXCEPTIONS` option sounds like it should be on or off and not about what kind of non-leagcy exception we choose) or adding another option?

Also I don't have any idea how widely this feature has been adopted in the first place. I haven't received a single bug reports about this feature over the last year... So I guess this is at least not widely used, if ever used anywhere.

Reply:
Yeah, I'm sure it's the case that nobody has been using it. I think Firefox is the only runtime that supports it so far, so nobody would really have a good reason to, when everything supports legacy exceptions.
As for options, I don't think it matters much. I'm imagining this would basically be an undocumented debugging option that we'd eventually get rid of once we're confident that LLVM and Binaryen's exnref support both work.

Reply:
Hmm, come to think of it, I don't think it will be very useful after all... We, or more likely I, will be the only one who will run that option (for debugging, in case llvm implementation has bugs or something), and I still can do that with `wasm-opt --translate-to-exnref` on the legacy result.. It's only one command and it's gonna be needed only when debugging, so I don't feel that's too inconvenient. And I'd like to avoid adding one more option to the soup of EH/SjLj options...

Reply:
Ah that's a good point that if it's a one-command translation that would be easy enough. Is the wasm/JS interface 100% compatible?

Reply:
(even if not, if we decide to do it, a followup is fine)

Reply:
> Ah that's a good point that if it's a one-command translation that would be easy enough. Is the wasm/JS interface 100% compatible?

Yeah I don't see why not. I checked with a simple test case, replacing only the wasm with the translated binary, and it seems to work.